server_code/authorisation.py

```diff
@@ -7,9 +7,28 @@
 import functools
 
 import anvil.users
+from anvil.tables import app_tables
 
 __version__ = "2.6.0"
 
+config = {"get_roles_row": lambda user: app_tables.users.get(user=user)["roles"]}
```

Comment:
This isn't right as the default option - the users table doesn't have a user column